app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] I recommend making the method private like in the other models.

Reply:
Solved :+1: